go.mod

```diff
@@ -4,11 +4,41 @@ go 1.14
 
 require (
 	github.com/edwarnicke/exechelper v1.0.1
+	github.com/ghodss/yaml v1.0.0
+	github.com/networkservicemesh/sdk v0.0.0-20200929140042-18220422e199
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.6.1
 	k8s.io/api v0.17.0
 	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v0.17.0
-	sigs.k8s.io/yaml v1.1.0
+	k8s.io/kubernetes v1.18.1
+	sigs.k8s.io/yaml v1.2.0
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.17.0
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.17.0
+	k8s.io/apimachinery => k8s.io/apimachinery v0.17.0
+	k8s.io/apiserver => k8s.io/apiserver v0.17.0
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.17.0
+	k8s.io/client-go => k8s.io/client-go v0.17.0
+	k8s.io/cloud-provider => k8s.io/cloud-provider v0.17.0
+	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.17.0
+	k8s.io/code-generator => k8s.io/code-generator v0.17.0
+	k8s.io/component-base => k8s.io/component-base v0.17.0
+	k8s.io/cri-api => k8s.io/cri-api v0.17.0
+	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.17.0
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.17.0
+	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.17.0
+	k8s.io/kube-proxy => k8s.io/kube-proxy v0.17.0
+	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.17.0
+	k8s.io/kubectl => k8s.io/kubectl v0.17.0
+	k8s.io/kubelet => k8s.io/kubelet v0.17.0
+	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.17.0
+	k8s.io/metrics => k8s.io/metrics v0.17.0
+	k8s.io/node-api => k8s.io/node-api v0.17.1
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.17.0
+	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.17.0
+	k8s.io/sample-controller => k8s.io/sample-controller v0.17.0
```

Comment:
Why do we need this?

Reply:
We needed this block for ClusterConfiguration from `k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta2` correct work. Replaced `k8s.io/kubernetes` with `sigs.k8s.io/cluster-api v0.3.10`